### PR TITLE
[EDU-3404] feat: add waf rule set

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Moved tables from WAF reference to WAF Rules Sets reference.
* Adjusted texts to make sense with new organization.
* Added more reference in to WAF Rules Sets reference.
* Updated sections for Rules Engine and Real-Time Metrics in WAF Rules Sets reference.

[EDU-3404]: https://aziontech.atlassian.net/browse/EDU-3404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 1]
```diff
@@ -1,47 +1,75 @@
...
+[Web Application Firewall (WAF)](/en/documentation/products/secure/edge-firewall/web-application-firewall/) provides the creation of rule sets to protect your edge applications against threat families.
 
-These rules are composed of criteria and behaviors. The criteria represent the conditions for executing the rules, and the behaviors represent the actions that need to be executed. The rules are processed sequentially, and if the conditions are met, the behaviors of each rule are executed. **WAF Rule Set** is what a set of custom rules for Web Application Firewall is called.
+Each threat receives a score, and it’ll be processed according to the sensitivity level set. If there’s more than one case for the same threat type, score will increase. 
 
-## Prerequisites
+After creating a rule set, you also need to create a rule in Rules Engine for Edge Firewall to execute the criteria and behavior.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:15]:
```suggestion
After creating a rule set, you also need to create a rule in [Rules Engine for Edge Firewall](/en/documentation/products/secure/edge-firewall/rules-engine/) to execute the criteria and behavior.
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 2]
```diff
@@ -1,47 +1,75 @@
...
+## Sensitivity levels
 
-- Regular Requests: all HTTP and HTTPS requests are analyzed by WAF and are considered secure.
-- Threats: the volume of threats detected by WAF and accounted for, when in *counting* mode. These threats aren't being blocked at the moment.
-- Threats Blocked: threats effectively blocked by WAF. To start blocking the threats found, the rule set must be in *blocking* mode.
+A request will be blocked by WAF if it obtains a score greater than or equal to the configured sensitivity level threshold. You can set one sensitivity level for each threat family.
 
+| Sensitivity | Description and WAF sore threshold |
+| ----------- | ---------------------------------- |
+| Lowest | The requisition will be considered a threat if it presents very strong evidence and receives a *score* greater than or equal to 40. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:48]:
```suggestion
| Lowest | The request will be considered a threat if it presents very strong evidence and receives a *score* greater than or equal to 40. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 3]
```diff
@@ -1,47 +1,75 @@
...
+| Low | The request will be considered a threat if it presents very strong evidence and receives a *score* greater than or equal to 24. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:49]:
```suggestion
| Low | The request will be considered a threat if it presents very strong evidence and receives a score greater than or equal to 24. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 4]
```diff
@@ -1,47 +1,75 @@
...
 ---
 
 import Button from '~/components/Button.astro'
 
-The **Rules Engine for Web Application Firewall (WAF)** is a feature developed to help you protect your edge applications in the context of your own applications. With Rules Engine for Web Application Firewall, you can define your own set of security rules (**Rule Set**), designed specifically for your needs. 
+[Web Application Firewall (WAF)](/en/documentation/products/secure/edge-firewall/web-application-firewall/) provides the creation of rule sets to protect your edge applications against threat families.
 
-These rules are composed of criteria and behaviors. The criteria represent the conditions for executing the rules, and the behaviors represent the actions that need to be executed. The rules are processed sequentially, and if the conditions are met, the behaviors of each rule are executed. **WAF Rule Set** is what a set of custom rules for Web Application Firewall is called.
+Each threat receives a score, and it’ll be processed according to the sensitivity level set. If there’s more than one case for the same threat type, score will increase.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:13]:
```suggestion
Each threat receives a score and it's processed according to the sensitivity level set. If there’s more than one case for the same threat type, the score will increase. 
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:15]:
```suggestion
After creating a rule set, you also need to create a rule in **Rules Engine for Edge Firewall** to execute the criteria and behavior.
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:48]:
```suggestion
| Lowest | The requisition will be considered a threat if it presents very strong evidence and receives a *score* equal to or greater than 40. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx (hunk 3 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:49]:
```suggestion
| Low | The request will be considered a threat if it presents very strong evidence and receives a *score* equal to or greater than 24. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 5]
```diff
@@ -1,47 +1,75 @@
...
-- Threats Blocked: threats effectively blocked by WAF. To start blocking the threats found, the rule set must be in *blocking* mode.
+A request will be blocked by WAF if it obtains a score greater than or equal to the configured sensitivity level threshold. You can set one sensitivity level for each threat family.
 
+| Sensitivity | Description and WAF sore threshold |
+| ----------- | ---------------------------------- |
+| Lowest | The requisition will be considered a threat if it presents very strong evidence and receives a *score* greater than or equal to 40. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
+| Low | The request will be considered a threat if it presents very strong evidence and receives a *score* greater than or equal to 24. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
+| Medium | Recommended sensitivity level. The request will be considered a threat if it presents sufficient evidence and receives a *score* greater than or equal to 16. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:50]:
```suggestion
| Medium | Recommended sensitivity level. The request will be considered a threat if it presents sufficient evidence and receives a *score* equal to or greater than 16. |
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 6]
```diff
@@ -1,47 +1,75 @@
...
+| High | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* greater than or equal to 8. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage on the variability of scenarios and uses of its application. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:51]:
```suggestion
| High | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* equal to or greater than 8. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage of the variability of scenarios and uses of its application. |
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 7]
```diff
@@ -1,47 +1,75 @@
...
+| Highest | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* greater than or equal to 4. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage on the variability of scenarios and uses of its application. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:52]:
```suggestion
| Highest | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score*equal to or greater than 4. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage of the variability of scenarios and uses of its application. |
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 8]
```diff
@@ -1,47 +1,75 @@
...
-```
+## Threat monitoring with Real-Time Metrics
 
-Based on this information, you can adjust the sensitivity of the *WAF Rule Set*, until no more false positives occur. You can also ask Azion to generate an *allowlist* for your application.
+Once you've completed your WAF rule set configuration and your edge application is receiving incoming traffic, you can use **Real-Time Metrics** to monitor requests and threats. You'll find charts comparing how WAF processed requests and a few specific charts for differente threat families.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:68]:
```suggestion
Once you've completed your WAF rule set configuration and your edge application is receiving incoming traffic, you can use **Real-Time Metrics** to monitor requests and threats. You'll find charts comparing how WAF processed requests and a few specific charts for different threat families.
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 9]
```diff
@@ -1,46 +1,77 @@
...
+| Scope | Source |
+| ----- | ------ |
+| Rule set | [How to create a WAF rule set](/en/documentation/products/guides/secure/create-waf-rule-set/) |
 
-Deixe a **WAF Rule Set** em **Counting Mode** pelo tempo que julgar necessário para que a maior parte das funcionalidades de sua aplicação sejam cobertas. 
+---
+
+## Tipos de ameaças
+
+A tabela **Threat Type Configuration** está disponível na [aba Main Settings](/pt-br/documentacao/produtos/edge-firewall/web-application-firewall/#waf-main-settings) de uma configuração de WAF. As ameaças são categorizadas em famílias, conforme o objetivo do ataque.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:28]:
```suggestion
A tabela **Threat Type Configuration** está disponível na aba [Main Settings](/pt-br/documentacao/produtos/edge-firewall/web-application-firewall/#waf-main-settings) de uma configuração de WAF. As ameaças são categorizadas em famílias, conforme o objetivo do ataque.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 10]
```diff
@@ -1,46 +1,77 @@
 ---
 title: WAF Rule Sets
-description: WAF Rule Sets
+description: Defina suas rule sets de proteção contra ameaças com o Azion Web Application Firewall (WAF).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:3]:
```suggestion
description: Defina suas rule sets de proteção contra ameaças com o Web Application Firewall (WAF) da Azion.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 11]
```diff
@@ -1,46 +1,77 @@
...
-Para configurar um **WAF Rule Set**, que é como uma configuração de WAF é chamada, você precisa ter uma configuração de Edge Firewall com o módulo **Web Application Firewall** ativado.
+Após criar um rule set, você também deve criar uma regra no **Rules Engine for Edge Firewall** para executar os critérios e comportamentos.
 
-:::tip[dica]
-Saiba mais sobre os [módulos do Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/) e o [Rules Engine para o Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/rules-engine/).
-:::
+## Implementação
 
-## Monitorando detecção de ameaças com Real-Time Metrics
+| Scope | Source |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:20]:
```suggestion
| Escopo | Recurso |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 12]
```diff
@@ -1,46 +1,77 @@
...
+| ----- | ------ |
+| Rule set | [How to create a WAF rule set](/en/documentation/products/guides/secure/create-waf-rule-set/) |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:22]:
```suggestion
| Rule set | [Como criar um WAF rule set](/pt-br/documentacao/produtos/guias/secure/criar-waf-rule-set/) |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 13]
```diff
@@ -1,46 +1,77 @@
...
+| Evading Tricks | Protege contra alguns truques de codificação utilizados para tentar escapar dos mecanismos de proteção. |
+| Unwanted Access | Detecta as tentativas de acesso a páginas administrativas ou vulneráveis, bots e ferramentas de *scanning* de segurança. |
+| Identified Attacks | Previne vários tipos de ataques comuns e vulnerabilidades conhecidas que certamente deverão ser bloqueados. |
+
+---
+
+## Níveis de sensibilidade
 
-No **Real-Time Metrics**, o primeiro gráfico da aba (threats vs requests) apresenta três séries temporais:
+Uma requisição será bloqueada caso ela obtenha um score do WAF maior ou igual ao limiar do nível de sensibilidade configurado. Você pode definir um nível de sensibilidade para cada família de ameaças.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:45]:
```suggestion
Uma requisição será bloqueada caso ela obtenha um score do WAF maior ou igual ao limite do nível de sensibilidade configurado. Você pode definir um nível de sensibilidade para cada família de ameaças.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 14]
```diff
@@ -1,46 +1,77 @@
...
+| Medium | A requisição será considerada uma ameaça se apresentar indícios suficientes e receber um *score* do WAF igual ou maior a 16. |
+| High | Ao menor indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* do WAF igual ou maior a 8. Esse nível de sensibilidade pode apresentar mais falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação. |
+| Highest | Ao mínimo indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* igual ou maior a 4. Esse nível de sensibilidade pode apresentar muitos falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação. |
 
-<Button href="/pt-br/documentacao/produtos/real-time-metrics/" text="saiba mais sobre Real-Time Metrics" variant="secondary"></Button>
+---
+
+## Configuração de Rules Engine
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:58]:
```suggestion
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 15]
```diff
@@ -1,46 +1,77 @@
...
-```
+## Monitoramento de ameaças com Real-Time Metrics
 
-Com base nessas informações, você pode ajustar a sensibilidade da *WAF Rule Set*, até que não ocorram mais falsos positivos. Você também pode solicitar para a Azion a geração de uma *allowlist* para sua aplicação.
+Após concluir a configuração de rule sets do seu WAF e sua edge application receber tráfego de entrada, você pode usar **Real-Time Metrics** para monitorar requisições e ameaças. Você encontrará gráficos comparando como o WAF processou as requisições e alguns gráficos específicos para diferentes famílias de ameaças.
 
+<Button href="/pt-br/documentacao/produtos/real-time-metrics/#waf" text="saiba mais sobre real-time metrics" variant="secondary"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:72]: URLs não levam o /observe/?

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/web-application-firewall.mdx [hunk 16]
```diff
@@ -33,34 +33,21 @@ Saiba mais sobre os [módulos do Edge Firewall](/pt-br/documentacao/produtos/edg
...
-| Highest       | Ao mínimo indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* maior ou igual a 4. Esse nível de sensibilidade pode apresentar muitos falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação.     |
-
-Cada nível de sensibilidade tolera um número definido de indícios de ameaças, o score do WAF representa essa quantidade de indícios de ameaças. Por isso, quanto mais flexível é o nível de sensibilidade, maior é o score do WAF que ela aceita. Já as sensibilidades mais rígidas, aceitam apenas requisições que somaram poucos indícios de ameaças.
+A seção de **Main Settings** é configurada ativando a proteção contra famílias específicas de ameaças: SQL Injection, Remote File Inclusions (RFI), Directory Traversal, Cross-Site Scripting (XSS), File upload, Evading tricks, Unwanted Access e Identified Attack,, e definindo um nível de sensibilidade: Lowest, Low, Medium, High, and Highest.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/web-application-firewall.mdx:36]:
```suggestion
A seção de **Main Settings** é configurada ativando a proteção contra famílias específicas de ameaças: SQL Injection, Remote File Inclusions (RFI), Directory Traversal, Cross-Site Scripting (XSS), File upload, Evading tricks, Unwanted Access e Identified Attack, e definindo um nível de sensibilidade: Lowest, Low, Medium, High, and Highest.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 17]
```diff
@@ -1,46 +1,77 @@
...
+| Low | A requisição será considerada uma ameaça se apresentar indícios muito fortes e receber um *score* do WAF igual ou maior a 24. Essa sensibilidade tem um menor nível de proteção para suas aplicações, mas também evitará o bloqueio de requisições com menor chance de representar ameaças (falsos positivos). |
+| Medium | A requisição será considerada uma ameaça se apresentar indícios suficientes e receber um *score* do WAF igual ou maior a 16. |
+| High | Ao menor indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* do WAF igual ou maior a 8. Esse nível de sensibilidade pode apresentar mais falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação. |
+| Highest | Ao mínimo indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* igual ou maior a 4. Esse nível de sensibilidade pode apresentar muitos falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:53]:
```suggestion
| Lowest | A requisição será considerada uma ameaça se apresentar indícios muito fortes e receber um *score* do WAF maior ou igual a 40. Essa sensibilidade tem um menor nível de proteção para suas aplicações, mas também evitará o bloqueio de requisições com menor chance de representar ameaças (falsos positivos) |
| Low | A requisição será considerada uma ameaça se apresentar indícios muito fortes e receber um *score* do WAF igual ou maior a 24. Essa sensibilidade tem um menor nível de proteção para suas aplicações, mas também evitará o bloqueio de requisições com menor chance de representar ameaças (falsos positivos) |
| Medium | A requisição será considerada uma ameaça se apresentar indícios suficientes e receber um *score* do WAF igual ou maior a 16 |
| High | Ao menor indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* do WAF igual ou maior a 8. Esse nível de sensibilidade pode apresentar mais falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação |
| Highest | Ao mínimo indício de uma ameaça, a requisição poderá ser bloqueada, mesmo quando apresentar um *score* igual ou maior a 4. Esse nível de sensibilidade pode apresentar muitos falsos positivos se a etapa de aprendizagem não tiver cobertura suficiente sobre a variabilidade de cenários e usos de sua aplicação |
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx [hunk 18]
```diff
@@ -1,47 +1,75 @@
...
+| Low | The request will be considered a threat if it presents very strong evidence and receives a *score* equal to or greater than 24. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives. |
+| Medium | Recommended sensitivity level. The request will be considered a threat if it presents sufficient evidence and receives a *score* equal to or greater than 16. |
+| High | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* equal to or greater than 8. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage of the variability of scenarios and uses of its application. |
+| Highest | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* equal to or greater than 4. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage of the variability of scenarios and uses of its application. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/waf-rule-sets.mdx:52]:
```suggestion
| Lowest | The request will be considered a threat if it presents very strong evidence and receives a *score* equal to or greater than 40. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives |
| Low | The request will be considered a threat if it presents very strong evidence and receives a *score* equal to or greater than 24. This sensitivity has a lower level of protection for your applications, but it'll also avoid blocking requests with less chance of false positives |
| Medium | Recommended sensitivity level. The request will be considered a threat if it presents sufficient evidence and receives a *score* equal to or greater than 16 |
| High | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* equal to or greater than 8. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage of the variability of scenarios and uses of its application |
| Highest | At the slightest hint of a threat, the requisition may be blocked, even when it has a *score* equal to or greater than 4. This level of sensitivity may present more false positives if the learning stage doesn't have sufficient coverage of the variability of scenarios and uses of its application |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 19]
```diff
@@ -1,46 +1,77 @@
...
+
+## Níveis de sensibilidade
 
-No **Real-Time Metrics**, o primeiro gráfico da aba (threats vs requests) apresenta três séries temporais:
+Uma requisição será bloqueada caso ela obtenha um score do WAF maior ou igual ao limiar do nível de sensibilidade configurado. Você pode definir um nível de sensibilidade para cada família de ameaças.
 
-- Regular Requests: todas as requisições HTTP e HTTPS analisadas pelo WAF e consideradas seguras.
-- Threats: o volume de ameaças detectadas pelo WAF e contabilizadas, quando em modo *counting*. Essas ameaças não estão sendo bloqueadas nesse momento.
-- Threats Blocked: ameaças efetivamente bloqueadas pelo WAF. Para começar a bloquear as ameaças encontradas, a rule set tem que estar em modo *blocking*.
+| Sensibilidade | Descrição e limiar do score |
```
[the remark this answers is not in the document]
Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:47]: Conferir com EN (sore/score)

**Review by MarianaAguilera — COMMENTED** [review]
Deixei algumas sugestões de estilo e um comentário

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx [hunk 20]
```diff
@@ -1,46 +1,76 @@
...
+| Unwanted Access | Detecta as tentativas de acesso a páginas administrativas ou vulneráveis, bots e ferramentas de *scanning* de segurança |
+| Identified Attacks | Previne vários tipos de ataques comuns e vulnerabilidades conhecidas que certamente deverão ser bloqueados |
+
+---
 
-<Button href="/pt-br/documentacao/produtos/real-time-metrics/" text="saiba mais sobre Real-Time Metrics" variant="secondary"></Button>
+## Níveis de sensibilidade
 
-Se você possuir também o serviço **Data Streaming**, é possível acompanhar informações mais detalhadas sobre IP, data e horário de acesso, status code, família de ataque detectada e modo de atuação configurado.
+Uma requisição será bloqueada caso ela obtenha um score do WAF maior ou igual ao limite do nível de sensibilidade configurado. Você pode definir um nível de sensibilidade para cada família de ameaças.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/waf-rule-sets.mdx:45]:
```suggestion
Uma requisição será bloqueada caso ela obtenha um score do WAF maior ou igual ao score do nível de sensibilidade configurado. Você pode definir um nível de sensibilidade para cada família de ameaças.
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/web-application-firewall.mdx [hunk 21]
```diff
@@ -33,34 +33,21 @@ Learn more about [Edge Firewall modules](/en/documentation/products/edge-firewal
...
+
+<Button href="/en/documentation/products/secure/edge-firewall/web-application-firewall/rules-engine/#threat-types" text="go to types of threats reference" variant="secondary"></Button> <br></br>
+
+<Button href="/en/documentation/products/guides/secure/create-waf-rule-set/" text="go to create a rule set guide" variant="secondary"></Button>
+
+It's possible to enable and disable protection for each threat family individually through the **Active** switch.
+
+**Sensitivity** levels define how strictly WAF will consider a request as a threat. 
+
+Each sensitivity level tolerates a defined number of threat indicatives and the WAF score represents this amount of threat indicatives. The more flexible the sensitivity level is, the higher the WAF score it accepts. The most rigid sensitivity levels, on the other hand, only accept requests with a fewer signs of threats.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/web-application-firewall/web-application-firewall.mdx:48]:
```suggestion
Each sensitivity level tolerates a defined number of threat indicatives and the WAF score represents this amount of threat indicatives. The more flexible the sensitivity level is, the higher the WAF score it accepts. The most rigid sensitivity levels, on the other hand, only accept requests with fewer signs of threats.
```

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**